cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/heartbeat/NotificationHeartbeatPublisher.java

```diff
@@ -37,7 +37,7 @@
   public NotificationHeartbeatPublisher(NotificationService notificationService) {
     this.notificationService = notificationService;
     this.heartbeatFeed = new Id.NotificationFeed.Builder()
-      .setNamespaceId(Constants.DEFAULT_NAMESPACE)
+      .setNamespaceId(Constants.SYSTEM_NAMESPACE)
```

Comment:
Always ``SYSTEM_NAMESPACE``? I think I get why, but want to confirm. 

Also, use ``Constants.SYSTEM_NAMESPACE_ID`` to avoid creating another object